overlays/holo-nixpkgs/hpos-holochain-api/src/index.js

```diff
@@ -24,19 +24,35 @@ app.get('/hosted_happs', async (_, res) => {
   }
   const presentedHapps = []
   for(let i=0; i < happs.length; i++) {
-    let enabled, source_chain;
+    let app_stats, enabled, source_chains, duration, bandwidth, cpu;
```

Comment:
```suggestion
    let app_stats, enabled, source_chains, duration, bandwidth, cpu, usage;
```

all four suggestions go together

Reply:
Just realized this is still a draft, so feel free to ignore my meddling ;)

Reply:
See the first comment. We're on the same page. :D